src/masks/InteractionMasks.tsx

```diff
@@ -139,24 +132,14 @@ export default function InteractionMasks<R, K extends keyof R>({
     return eventBus.subscribe(EventTypes.DRAG_ENTER, handleDragEnter);
   }, [draggedPosition, eventBus]);
 
-  const editorPosition = useMemo(() => {
-    if (!isEditorEnabled || !selectionMaskRef.current) return null;
-    const { left: selectionMaskLeft, top: selectionMaskTop } = selectionMaskRef.current.getBoundingClientRect();
-    if (editorPortalTarget === document.body) {
-      const { scrollLeft, scrollTop } = document.scrollingElement || document.documentElement;
-      return {
-        left: selectionMaskLeft + scrollLeft,
-        top: selectionMaskTop + scrollTop
-      };
-    }
-
-    const { left: portalTargetLeft, top: portalTargetTop } = editorPortalTarget.getBoundingClientRect();
-    const { scrollLeft, scrollTop } = editorPortalTarget;
+  function getEditorPosition() {
+    if (!canvas.current) return null;
+    const { left, top } = canvas.current.getBoundingClientRect();
     return {
-      left: selectionMaskLeft - portalTargetLeft + scrollLeft,
-      top: selectionMaskTop - portalTargetTop + scrollTop
+      left: columns[selectedPosition.idx].left - scrollLeft + left,
```

Comment:
Should we remove `editorPortalTarget` prop?

Reply:
nm, it is needed for the portals

Reply:
I guess we could, do we have a need for it anymore?

Reply:
I am not too sure actually 🤔